### ISSUE TITLE
Default theme and export partials are not created (Umbraco Forms 10.0.0-rc2)

### ISSUE THREAD

**Issue by argentini** [umbraco/Umbraco.Forms.Issues#787]
The form theme picker property editor still points to the Views/Partials/Forms path, yet Umbraco Forms does not put any default theme partial(s) in there, nor does it add the default export partial(s).

### Specifics

Using Umbraco Forms 10.0.0-rc2 in Umbraco CMS 10.0.0-rc5

### Expected result

Forms default view partials should be created.

### Actual result

Forms default view partials are not created.

**Comment by AndyButland** [comment]
Thanks for sharing details of this.  It's actually correct that these files don't exist on disk, as they are now served by a razor class library and hence are embedded resources within one of the dlls.

But we have missed an update necessary to the theme picker that you've noted here, so we'll get that fixed up for the full release next week.

**Comment by argentini** [comment]
@AndyButland thanks for the info. The default theme and export code has always been helpful for creating custom variations. If these assets are embedded that won't be possible, at least in development. When you publish the Umbraco site those embedded assets do get created as physical files. Those files are stored in a new location, `wwwroot/App_Plugins/UmbracoForms`. So changes to the form theme property editor should work with existing paths as well as this new path.

**Comment by AndyButland** [comment]
Yes, that should be the case with the new setup.  The theming system will look for files either from the embedded ones shipped with the product, or the old location on disk.  So custom themes created as before should work.

We changed this to ship them embedded using a razor class library, as although the files were on disk when you installed the Forms package in Umbraco 9, they weren't "really" editable.  I.e. you could change them, but with a `dotnet clean` they would get removed, and changes would get lost.  So we thought it was more explicit to not ship with these files that look like they can be changed.

Now the files aren't on disk it's not quite so easy to refer to them of course, when creating a custom theme.  But there's [reference in the documentation](https://our.umbraco.com/documentation/add-ons/umbracoforms/developer/Themes/#creating-a-theme) to a zip file that you can download to review and copy files from the default theme.

If you'd like to check it, I've just [released a new RC of Forms](https://www.nuget.org/packages/Umbraco.Forms/10.0.0-rc3) that fixes the issue you found here, so hopefully if you upgrade to that, you should find you can create and use custom themes as before.  And if course, if you have any trouble, please let us know and we'll get it sorted.

**Comment by argentini** [comment]
@AndyButland I just grabbed RC3, so thanks for that :)

**Comment by hfloyd** [comment]
@AndyButland - Related to this:

> Now the files aren't on disk it's not quite so easy to refer to them of course, when creating a custom theme. But there's 
[reference in the documentation](https://our.umbraco.com/documentation/add-ons/umbracoforms/developer/Themes/#creating-a-theme) to a zip file that you can download to review and copy files from the default theme.

Is there any way to link the [v10 documentation](https://docs.umbraco.com/umbraco-forms/developer/themes) out to a directory (like the umbraco nightlies) so it would be possible to see the latest available, and match the downloaded files with the in-use version of Forms on your project?

**Comment by AndyButland** [comment]
I've got a couple of docs amends pending to help make this a little more visible:

- Short links to the latest versions for each Umbraco major.
- Each minor version will have a file downloadable from the "Source Code" tab at the bottom of [the package page on our](https://our.umbraco.com/packages/developer-tools/umbraco-forms/).

This is what will be added to the docs page that you linked:

```
## Creating a Theme

To create a theme, you need to create a folder at `/Views/Partials/Forms/Themes/`. The name of the folder is the name of theme that will be visible in the backoffice when choosing it.

Copy the explicit files you wish to override in your theme, it may be a single file or all files from the `default` theme folder. Make the necessary changes you desire to CSS class names, markup etc.

### Obtaining the Default Theme Files

For Umbraco 9 and previous, it's straightforward to simply copy the files you need from the default theme folder. We highly recommend you never customize any files found in the `default` themes folder, as any customizations to these files will most likely be lost with any future upgrades you do to Umbraco Forms.

Umbraco 10+ distributes these files as part of a Razor class library, so you won't find them on disk. The easiest way to obtain copies of the files included in the default theme is to [download this zip file](https://our.umbraco.com/FileDownload?id=23911) and extract the ones you need.

You can obtain the latest versions of the default theme for each Umbraco major version from the following links:

- [Umbraco 10 Default Theme](https://umbra.co/umbraco-forms-default-theme)
- [Umbraco 11 Default Theme](https://umbra.co/umbraco-forms-default-theme-11)

The default theme as of each minor version is available from the "Source Code" tab available at the bottom of [this page](https://our.umbraco.com/packages/developer-tools/umbraco-forms/). Only versions where changes were made are available. So if you are running for example Umbraco 11.1, and no file for that version is available, you can reference the one for Umbraco 11.0.
```
